src/app/components/search-user/search-user.component.html

```diff
@@ -3,10 +3,10 @@
   #searchBar
     type="text"
     placeholder="Search"
-    id="searchbox"
     (keyup.enter)="onSearch(searchBar.value)"
+    class="form-control"
     />
-  <button
-    type="button" (click)="onSearch(searchBar.value)">Search
-  </button>
 </div>
+
```

Comment:
There shouldn't be any unnecessary empty spaces

Reply:
Sorry,I'll fix all the unnecessary spaces and raise a pr again!. Linter didn't show me any error so I ignored that.

Reply:
No need to raise another PR. Just add commits to your branch and push it. The PR will update automatically.

Reply:
Ok Sure

Reply:
components do not exist